Dockerfile

```diff
@@ -1,5 +1,10 @@
+# Golang image and version (defaults are provided).
+# Use e.g. `golang` for multi-arch support.
+ARG IMG_NAME
+ARG IMG_VERSION
+
 # Build the manager binary
-FROM quay.io/bitnami/golang:1.16 as builder
+FROM ${IMG_NAME:-quay.io/bitnami/golang}:${IMG_VERSION:-1.16} as builder
```

Comment:
This makes me wonder if cc needs it's own [versions database](https://github.com/kata-containers/kata-containers/wiki/Glossary#versions-database) for scenarios like this.

Reply:
Hmm, `yq`ing a version from _Dockerfile_ seems tricky at least

Reply:
https://github.com/kata-containers/kata-containers/issues/3162 :smile: